app/data/census_individual.json

```diff
@@ -160,11 +79,11 @@
                 },
                 {
                     "id": "date-of-birth",
-                    "title": "John Doe",
+                    "title": "{{ answers.person_full_name }}",
```

Comment:
Sorry, I meant if we used it for something like the HTML title in a future version. I think the title should be the title for the page, so for this block it would be 'Date of birth'.